src/Interop/src/Hook.cpp

```diff
@@ -156,7 +156,7 @@ namespace OpenLoco::Interop
         if (!done)
         {
             const auto errCode = static_cast<uint32_t>(GetLastError());
-            fprintf(stderr, "WriteProcessMemory failed! address = 0x%08x, size = %d, GetLastError() = 0x%08x", address, i, errCode);
+            fprintf(stderr, "WriteProcessMemory failed! address = 0x%08x, size = %d, GetLastError() = 0x%08x\n", address, i, errCode);
```

Comment:
Good opportunity to change this to Logging::err and use the correct format which is the same std::format

Reply:
No need to address this in this PR.